modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -536,4 +563,49 @@
       end
     end
   end
+
+  describe 'health status information in edit page' do
+    frozen_date_time = Time.current
```

Comment:
```suggestion
    frozen_date_time = Time.at(1_652_111_317)
```

Then we have a consistent timestamp